db/seeds.rb

```diff
@@ -93,13 +112,20 @@
                   tenant_id: bridge_builders1.id
                   )
 
+timmys_vaccines_nigeria.photos << Photo.create!(
+                  image: File.new( "#{Rails.root}/app/assets/images/timmys_vaccines.jpg")
+                  )
+
 stevens_books_bangkok = Project.create!(
                   title: "Steven's school books",
                   price: 4000,
                   description: "How can I teach deez kiiiids?" * 5,
                   retired: false,
                   categories: [startup_category],
-                  tenant_id: bridge_builders1.id
+                  tenant_id: bridge_builders2.id
+                  )
+stevens_books_bangkok.photos << Photo.create!(
+                  image: File.new( "#{Rails.root}/app/assets/images/stevens_books.jpg" )
```

Comment:
Line is too long. [88/80]<br>Indent the first parameter one step more than the previous line.<br>Space inside parentheses detected.